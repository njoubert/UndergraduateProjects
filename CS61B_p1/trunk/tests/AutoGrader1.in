Sara lives around here.
Sara is the clown.
Sara lives in the dark house.
Sara lives in the dark house.
What do you know about Sara?
Who lives in the dark house?
Who is the clown?

