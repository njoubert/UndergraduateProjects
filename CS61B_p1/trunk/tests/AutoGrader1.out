1. Sara lives around here.
2. Sara is the clown.
3. Sara lives in the dark house.
4. Sara lives in the dark house.
Q: What do you know about Sara?
A: Sara is the clown and lives in the dark house.
Q: Who lives in the dark house?
A: Sara lives in the dark house.
Q: Who is the clown?
A: Sara is the clown.
